ftw/publisher/sender/workflows/contextstate.py

```diff
@@ -91,10 +91,16 @@ def get_unpublished_references(self):
         for obj in self.get_references():
             if obj is None:
                 continue
-
             obj_state = getMultiAdapter((obj, self.request),
                                         IPublisherContextState)
 
+            if (obj == self.context
+                or (not obj_state.has_workflow()
+                    and obj_state.get_parent_state().context == self.context)):
```

Comment:
Please use the utility function [`belongs_to_parent` (`ftw.publisher.core`)](https://github.com/4teamwork/ftw.publisher.core/blob/5e03c042e4d11560ad7bcb8fb4d4182dace83b9d/ftw/publisher/core/belongs_to_parent.py#L38), because this covers all use cases.

Reply:
👍 did not know that this existed thanks for pointing it out!

Reply:
I have updated the code to use `belongs_to_parent`.